syntax = "proto3";

package pblogger;

// Interface exported by the server. Used for sending code logs as well as telemetry
service Log {
  // Accept a stream of SingleLog and send a dummy message
  rpc SendSingleLog(SingleLog) returns (Empty) {}
}


message SingleLog {
  string logtype = 1;
  string logstring = 2;
}

message Empty {

}